crates/next-core/src/next_font_google/util.rs

```diff
@@ -94,8 +94,9 @@ pub(crate) fn get_font_axes(
                 variable_axes: Some(variable_axes),
             })
         }
+
         FontWeights::Fixed(weights) => Ok(FontAxes {
-            wght: weights.clone(),
+            wght: IndexSet::from_iter(weights.iter().map(|w| w.to_string())),
```

Comment:
Is the weights already sorted? Should this just be a vec instead?

Reply:
Yeah, these sets are only ever iterated (membership is never checked). I went with the IndexSet here because of the invariant that there can't be duplicated values. 

At this point `FontWeights` also being a set would take care of duplicates from the query string, which itself is derived from how the user invoked the font function. So in practice it's probably not an issue outside of programming errors. Does `AutoSet` preserve insertion order?